rasa/cli/export.py

```diff
@@ -173,7 +173,10 @@ def export_trackers(args: argparse.Namespace) -> None:
     Args:
         args: Command-line arguments to process.
     """
-    asyncio.run(_export_trackers(args))
+    # we can replace it with asyncio.run(..) but it doesn't work in Python 3.9
```

Comment:
can we move this to a util function so that we only have this segment in one place?